### ISSUE TITLE
Disagreement with styler and lintr about long lines. Rstudio gives a better format

### ISSUE THREAD

**Issue by maikol-solis** [r-lib/styler#1192]
Hi! 

I have this problem. Suppose you have this long line in your code:

```
value <- my_function(arg0 = 0, arg1 = 1, arg2 = 2, arg3 = 3, arg4 = 4, arg5 = 5, arg6 = 6, arg7 = 7)
```

Checking it with `styler`, returns the unmodified string. 
```r
styler::style_text("value <- my_function(arg0 = 0, arg1 = 1, arg2 = 2, arg3 = 3, arg4 = 4, arg5 = 5, arg6 = 6, arg7 = 7)")
value <- my_function(arg0 = 0, arg1 = 1, arg2 = 2, arg3 = 3, arg4 = 4, arg5 = 5, arg6 = 6, arg7 = 7)
```
However, `lintr` is complaining about the length of the line
```
[line_length_linter]: Lines should not be more than 80 characters. This line is 100 characters.	
```

In my opinion, Rstudio gives a better format: 
```r
value <-
  my_function(
    arg0 = 0,
    arg1 = 1,
    arg2 = 2,
    arg3 = 3,
    arg4 = 4,
    arg5 = 5,
    arg6 = 6,
    arg7 = 7
  )
```

What can I do to make `styler` behave like Rstudio in this case? 

Best.

**Comment by IndrajeetPatil** [comment]
Closing in favour of #247 

If you want to avoid the lint, you can change the length limit in the [lintr config](https://github.com/r-lib/lintr/blob/871fee3e4371a3b61bd0472ba3dbe493570328e1/.lintr#L3):

``` r
library(lintr)

lint(
  text = "value <- my_function(arg0 = 0, arg1 = 1, arg2 = 2, arg3 = 3, arg4 = 4, arg5 = 5, arg6 = 6, arg7 = 7)",
  linters = line_length_linter(120L)
)
```

<sup>Created on 2024-04-20 with [reprex v2.1.0](https://reprex.tidyverse.org)</sup>

**Comment by maikol-solis** [comment]
Thanks for the tip.
